litgpt/prompts.py

```diff
@@ -288,6 +288,11 @@ def apply(self, prompt: str, **kwargs: str) -> str:
         return f"<start_of_turn>user\n{prompt}<end_of_turn>\n<start_of_turn>model\n"
 
 
+class H2Oai(PromptStyle):
+    def apply(self, prompt: str, **kwargs: str) -> str:
+        return f"<s><|prompt|>{prompt}</s><|answer|>"
```

Comment:
```suggestion
        return f"<|prompt|>{prompt}</s><|answer|>"
```

Is the opening tag needed?

```python
from transformers import AutoTokenizer

tokenizer = AutoTokenizer.from_pretrained("h2oai/h2o-danube2-1.8b-chat")
messages = [
    {"role": "user", "content": "{prompt}"},
]
prompt = tokenizer.apply_chat_template(
    messages,
    tokenize=False,
    add_generation_prompt=True,
)
print(prompt)
>> "<|prompt|>{prompt}</s><|answer|>"
```

Reply:
I checked the template HF uses there opening tag is not used, so we can remove it. Thanks.